cvat/apps/engine/task.py

```diff
@@ -252,25 +253,42 @@ def update_progress(progress):
     compressed_chunk_writer = compressed_chunk_writer_class(db_data.image_quality)
     original_chunk_writer = original_chunk_writer_class(100)
 
+    # calculate chunk size if it isn't specified
+    if db_data.chunk_size is None:
+        if isinstance(compressed_chunk_writer, ZipCompressedChunkWriter):
+            w, h = extractor.get_image_size()
+            area = h * w
+            if area <= 1920 * 1080:
```

Comment:
Does it make sense to have for small resolutions bigger chunk? Something like `max(1920 * 1080 * 18 // area * 2, 2)`

Reply:
Maybe better to have `clip(1920 * 1080 * 18 // area * 2, 2, 72)`